examples/admin-partitions/terraform/variables.tf

```diff
@@ -20,7 +20,7 @@ variable "tags" {
 variable "consul_ecs_image" {
   description = "Consul ECS image to use."
   type        = string
-  default     = "hashicorpdev/consul-ecs:latest"
+  default     = "hashicorppreview/consul-ecs:0.7.0-rc1"
```

Comment:
can we define a variable 
```
variable "default_consul_ecs_image" {
  default = "hashicorppreview/consul-ecs:0.7.0-rc1"
}
```
and use it here like this - 
```
variable "consul_ecs_image" {
  description = "Consul ECS image to use."
  type        = string
  default     = "hashicorpdev/consul-ecs:latest"
  default     = var.default_consul_ecs_image
}
```
or this is not suitable here? I see this repeated lot of times so I am suggesting.

Reply:
The method you proposed won't work across submodules. In the current state, we define one variable per submodule/terraform example.

Reply:
ok got it.